lib/components/pwa-prompt.tsx

```diff
@@ -98,16 +70,7 @@ export const PwaPrompt = ({
     ]
   );
 
-  const aboveMinVisits = iosPwaPrompt.visits >= promptOnVisit;
```

Comment:
Probably want to do something here so the prompt can't show if `isOpen` is false.

Still need to figure out what's best in this case.